bookmarks/styles/bookmark-page.scss

```diff
@@ -122,6 +122,22 @@ li[ld-bookmark-item] {
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+
+    &[data-title]:hover::after {
```

Comment:
While this is a neat trick, it's super annoying that this immediately pops up when you hover over a link. 

I would be more open to using the regular `title` attribute.

Reply:
@sissbruecker I hear you. I force-pushed a new commit which makes two amendments. 

- The pop-up will only show if the text is partially hidden.
- There is a ~300ms delay before the pop-up shows up.

Reply:
Thanks. Did some tweaks:
- Restrict tooltip max width to bookmark item so longer titles don't overflow
- Make tooltip show up on focus, for example when using keyboard navigation
- Refactored JS so that it reuses existing bookmark item behavior, doesn't need to duplicate title into a data attribute
- Refactored CSS to reuse Spectre CSS variables